### PR TITLE
[IMP] account: Improve the validation process thanks to the "to check" boolean

### PR DESCRIPTION
**Pull request by jbw-odoo**
- keep track of to_check changes in chatter
- add "set as checked" button
- make the "to_check" field optional in list view
- vendor bill / customer invoice : remove "entry" from "cancel entry" button.

Task: 2389849

### CODE REVIEW

File: addons/account/data/account_data.xml [hunk 1]
```diff
@@ -114,6 +114,20 @@
...
+            <field name="hidden" eval="True"/>
+            <field name="description">Set as to check</field>
+        </record>
+        <record id="mt_invoice_to_check_false" model="mail.message.subtype">
+            <field name="name">Set as checked</field>
+            <field name="res_model">account.move</field>
+            <field name="default" eval="False"/>
+            <field name="hidden" eval="True"/>
+            <field name="description">Set as checked</field>
+        </record>
```
Comment (qdp-odoo) [inline addons/account/data/account_data.xml:130]: do we really need the new subtypes ??

**Comment by smetl** [comment]
@jbw-odoo Plz green ci/runbot :)

**Comment by jbw-odoo** [comment]
> @jbw-odoo Plz green ci/runbot :)

done :)

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**